packages/core/src/parsers/ComponentParser.js

```diff
@@ -614,6 +614,20 @@ class ComponentParser {
     return node;
   }
 
+  static convertMdExtToHtmlExt(renderedContent) {
+    const $ = cheerio.load(renderedContent);
```

Comment:
👀 

`cheerio.load` is really convenient but expensive (it parses the entire page one more time);

you can look at the other functions for how to manipulate the pseudo dom (which is also still usable with cheerio - so we don't sacrifice code quality for more complex operations) (but still you shouldn't need cheerio for the simple conversion here)

wondering if we should move this function to `linkProcessor` since it deals with links as well

Reply:
Yup, thanks for highlighting the expensive nature of ```cheerio.load```. 

I have modified the function to modify nodes directly as we are traversing through them in the ```_parse``` function. 

I have also shifted the function into `linkProcessor`!